File: ServerApp/Config/Objects/Docs/Declaration.cs
using LBTDTools.ServerApp.Config.Objects.Docs.Properties.Act;
using LBTDTools.ServerApp.Config.Objects.Items;
using LBTDTools.ServerApp.Config.Objects.Items.Car;

namespace LBTDTools.ServerApp.Config.Objects.Docs
{
    public class Declaration
    {
        public Laboratory Laboratory { get; set; }
        public Service Service { get; set; }
        public Car Car { get; set; }
        public Client Client { get; set; }
    }
}

File: ClientApp/Config/Objects/Docs/Declaration.ts
import Client from "../Items/Client";
import Laboratory from "../Items/Laboratory";
import Service from "../Items/Service";

export default class Declaration 
{
    public Laboratory: Laboratory;
    public Service: Service;
    public Client: Client;
    
    public WorksDate: string;

    constructor(laboratory: Laboratory, service: Service, client: Client, WorksDate: string) 
    {
        this.Laboratory = laboratory;
        this.Service = service;
        this.Client = client;
        
        this.WorksDate = WorksDate;
    }

}